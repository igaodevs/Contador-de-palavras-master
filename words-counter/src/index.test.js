import Index from './index.js';

describe('Renderizacao inicial', () => {
  it('Renderiza sem quebrar', () => {
    expect(
      JSON.stringify(
        Object.assign({}, Index, { _reactInternalInstance: 'testingAppRender' }),
      ),
    ).toMatchSnapshot();
})})