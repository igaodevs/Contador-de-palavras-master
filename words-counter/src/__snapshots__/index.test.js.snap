// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renderizacao inicial Renderiza sem quebrar 1`] = `"{\\"_reactInternalInstance\\":\\"testingAppRender\\"}"`;
